qemu/tests/hpt_miscellaneous.py

```diff
@@ -88,6 +103,16 @@ def run(test, params, env):
         # For HPT reszing after hotplug memory
         hpt_mem = MemoryHotplugTest(test, params, env)
         hpt_mem.hotplug_memory(vm, "hpt_mem")
+        orig_mem = int(session.cmd_output(cmd=params['free_mem_cmd']))
+        in_mem = int(
```

Comment:
`in_mem = int(` -> `increase_mem = int(`

Reply:
Updated it from in_mem to increase_mem.